src/interactivity/win32/Clipboard.cpp

```diff
@@ -208,10 +208,8 @@ void Clipboard::StoreSelectionToClipboard(bool const copyFormatting)
     const auto& gci = ServiceLocator::LocateGlobals().getConsoleInformation();
     const auto& buffer = gci.GetActiveOutputBuffer().GetTextBuffer();
 
-    const auto defaultForeground = gci.GetDefaultForeground();
-    const auto defaultBackground = gci.GetDefaultBackground();
     const auto GetAttributeColors = [=, &gci](const auto& attr) {
-        return gci.LookupAttributeColors(attr, defaultForeground, defaultBackground);
+        return gci.LookupAttributeColors(attr);
     };

```

Comment:
This is another case where we were attempting to optimise the `LookupAttributeColors` call, which is no longer necessary.